src/components/views/ProductListImages/ProductListImages.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ProductBox from '../../common/ProductBox/ProductBox';
+
+const ProductListImages = ({ id, products}) => {
+  return (
+    <div className='container'>
+      <div className='row'>
+        <div className='col-9'>
+          <div className='row'>
+            {products &&
+    products
+      .filter(item => item.category === id)
+      .map((product, i) => (
+        <div key={i} className='col-4'>
+          <ProductBox {...product} />
+        </div>
+      ))}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+ProductListImages.propTypes = {
+  category: PropTypes.string,
```

Comment:
Nie przekazujesz tutaj obiektu `category`  tylko jego właściwośći,co widać jak spojrzymy na kontener dla tego komponenetu (użyłaś operatora spread)

```
  return {
    ...category,
```

A skoro już mamy właściwości kateogri, to warto nad listą produktów przynajmniej wyświetlić w nagłówku `<h2>` tą nazwę kateogrii, pozowli nam to zweryfikowac na tą chwilę czy w ogóle dobrze się te kategorie przekazuję